text/0000-prepare-phase.md

```diff
@@ -0,0 +1,105 @@
+# Meta
+[meta]: #meta
+- Name: Prepare Phase
+- Start Date: 2021-01-13
```

Comment:
We could update this to be more recent I think.

Reply:
date is updated for today.